Packs/CommonScripts/.pack-ignore

```diff
@@ -73,6 +73,30 @@ ignore=BA119
 [file:HideFieldsOnNewIncident.yml]
 ignore=SC105
 
+[file:TAXII2_Performance_Test_-_Concurrency.yml]
```

Comment:
Why do we have the change here in CommonScripts? we don't touch this pack in this PR.
Please revert

Reply:
ok, reverting all changes in common scripts